index.html

```diff
@@ -5,37 +5,94 @@
+            <h2 class="home-footer">2015.10.15(THU) ~ 16(FRI)</h2>
+            <h3 class="home-copywrite">&copy; National Museum of Korea, Art Center Nabi and more</h3>
         </section>
         <section class="main-program">
-
+            <h2 class="main-program__title">Main Program</h2>
+            <span class="main-program__line"></span>
+            <ul class="main-program__container">
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/lecture.svg" alt="list_image_one" class="list-head__img">
+                        <h3>Lecture</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.</p>
+                </li>
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/icon2.svg" alt="list_image_two" class="list-head__img">
+                        <h3>CC Exhibition</h3>
+                    </div>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. </p>
+                </li>
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/forum.svg" alt="list_image_three" class="list-head__img">
+                        <h3>Forum</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium cum cumque quasi</p>
+                </li>
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/icon4.svg" alt="list_image_four" class="list-head__img">
+                        <h3>Workshop</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                </li>
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/icon5.svg" alt="list_image_five" class="list-head__img">
+                        <h3>CC Ignite</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Accusamus placeat ab unde reiciendis dolor.</p>
+                </li>
+            </ul>
+            <button class="main-program__button">Join CC Global Summit 2015</button>
         </section>
         <section class="featured-speakers">
-
-        </section>
-        <section class="partners">
-            
+            <h2 class="featured-speakers__title">Featured Speakers</h2>
+            <span class="featured-speakers__line"></span>
+            <ul class="featured-speakers__container"></ul>
+            <button class="featured-speakers__button">More</button>
         </section>
     </main>
     <footer>
 
     </footer>
```

Comment:
- [ ] While you make a great effort 😃 I see that you forgot to add the footer at the bottom of the page, so please try to add the content of the footer to make the design nice and stick to the Behance design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233068159-e26e5840-88cb-441f-a9b1-9ba1a41b0cee.png)   |  ![image](https://user-images.githubusercontent.com/83360636/233068305-3f21a2ed-91cb-4537-b57a-c140d2a89c98.png)  |